package.json

```diff
@@ -19,23 +20,23 @@
   },
   "homepage": "https://github.com/decentralized-identity/lds-ecdsa-secp256k1-2019.js#readme",
   "devDependencies": {
-    "bip39": "^2.5.0",
     "codecov": "^3.1.0",
     "coveralls": "^3.0.2",
-    "did-jwt": "^0.1.3",
-    "ethereumjs-util": "^6.0.0",
-    "hdkey": "^1.1.0",
     "jest": "^23.6.0"
   },
   "dependencies": {
     "base64url": "^3.0.1",
+    "bip39": "^2.5.0",
```

Comment:
I think these are actually dev dependencies, eslint likely complains about them. similarly eslint is a dev dependency.